crates/sui/src/unit_tests/keytool_tests.rs

```diff
@@ -195,6 +195,58 @@ async fn test_load_keystore_err() {
     assert!(FileBasedKeystore::new(&path2).is_err());
 }
 
+#[test]
+async fn test_private_keys_ed25519() -> Result<(), anyhow::Error> {
+    // private key, address
+    const TEST_CASES: [[&str; 3]; 3] = [
+        [
+            "0x9dd9ae36ee51b912a0364c58c1f21333bcdad2d91911aa127226c512be285102",
+            "AJ3ZrjbuUbkSoDZMWMHyEzO82tLZGRGqEnImxRK+KFEC",
+            "0x90f3e6d73b5730f16974f4df1d3441394ebae62186baf83608599f226455afa7",
+        ],
+        [
+            "0xeea84be738c59f56ee94dae8fd5a68082d4579ed38548d6ec4017da6c5619bf3",
+            "AO6oS+c4xZ9W7pTa6P1aaAgtRXntOFSNbsQBfabFYZvz",
+            "0xfd233cd9a5dd7e577f16fa523427c75fbc382af1583c39fdf1c6747d2ed807a3",
+        ],
+        [
+            "0x91e8808c489ee0cc99c2edf79e63be6a144f8f600c7411bc2e806f7255710674",
+            "AJHogIxInuDMmcLt955jvmoUT49gDHQRvC6Ab3JVcQZ0",
+            "0x81aaefa4a883e72e8b6ccd3bec307e25fe3d79b14e43b778695c55dcec42f4f0",
+        ],
+    ];
+    // assert correctness
+    for t in TEST_CASES {
```

Comment:
nit: You might be able to use a slice type to avoid having to always update the outer `3` in your `TEST_CASES` type, and you can also use tuples and tuple destructuring to give names to your various test case components.

Also, nit: I think the comment is out-of-date, as there's three components here.

```suggestion
    // private key, address
    const TEST_CASES: &[(&str, &str, &str)] = &[
        [
            "0x9dd9ae36ee51b912a0364c58c1f21333bcdad2d91911aa127226c512be285102",
            "AJ3ZrjbuUbkSoDZMWMHyEzO82tLZGRGqEnImxRK+KFEC",
            "0x90f3e6d73b5730f16974f4df1d3441394ebae62186baf83608599f226455afa7",
        ],
        [
            "0xeea84be738c59f56ee94dae8fd5a68082d4579ed38548d6ec4017da6c5619bf3",
            "AO6oS+c4xZ9W7pTa6P1aaAgtRXntOFSNbsQBfabFYZvz",
            "0xfd233cd9a5dd7e577f16fa523427c75fbc382af1583c39fdf1c6747d2ed807a3",
        ],
        [
            "0x91e8808c489ee0cc99c2edf79e63be6a144f8f600c7411bc2e806f7255710674",
            "AJHogIxInuDMmcLt955jvmoUT49gDHQRvC6Ab3JVcQZ0",
            "0x81aaefa4a883e72e8b6ccd3bec307e25fe3d79b14e43b778695c55dcec42f4f0",
        ],
    ];
    // assert correctness
    for (input_string, keypair, address)  in TEST_CASES {
```

Reply:
Fixed